hs_core/models.py

```diff
@@ -1185,9 +1186,8 @@ def bag_url(cls, resource_id):
     @classmethod
     def scimeta_url(cls, resource_id):
         scimeta_path = "{resource_id}/data/resourcemetadata.xml".format(resource_id=resource_id)
-        istorage = IrodsStorage()
-        scimeta_url = istorage.url(scimeta_path)
-
+        scimeta_url = reverse('rest_download',
+                          kwargs={'path': scimeta_path})
```

Comment:
@jbradberry Have removed ```hs_core/models.py``` as well as two files from ```django_irods``` submodule in setup.cfg and resolved all pylint complaints except the following two:
```
./hs_core/models.py:2265:1: E402 module level import not at top of file
./hs_core/models.py:2265:1: F401 'receivers' imported but unused
```
@pkdash You added the ```import receivers``` at the end of ```models.py``` with comments explaining why you have to do so to make receivers to work, so I cannot move it to the top of the module to comply with pylint rules. @jbradberry Can we configure something to ignore these two specific pylint complaints so that Jenkins will pass this PR build? @pkdash Do you think we can move this receivers import to the top of the module to comply with pylint rules? Anyway, @jbradberry @pkdash I have addressed all your concerns so far. Please see if you have further concerns and if you can give a +1.